### ISSUE TITLE
RemovedInDjango110Warning: SubfieldBase has been deprecated.

### ISSUE THREAD

**Issue by tom-dalton-fanduel** [omab/python-social-auth#997]
Using 0.2.21 with Django 1.9, I get the following deprecation warning:

`RemovedInDjango110Warning: SubfieldBase has been deprecated. Use Field.from_db_value instead.`

It looks like this was originally a problem and was fixed by #813. However, it looks like recently the warning was reintroduced, by this:

https://github.com/omab/python-social-auth/commit/03ae00154900a8150a5bf941663425698aacdb6b#diff-25f350079af94bedfb90b3a9e8cafb59R15

**Comment by tom-dalton-fanduel** [comment]
I wonder if the try/catch could be replaced with a version check along the lines of

```python
# SubfieldBase causes RemovedInDjango110Warning in 1.8 and 1.9, and will not work in 1.10 or later
if django.VERSION[:2] >= (1, 8):
    field_class = functools.partial(six.with_metaclass, type)
else:
    from django.db.models import SubfieldBase
    field_class = functools.partial(six.with_metaclass, SubfieldBase)
```

**Comment by ThibautNicodeme** [comment]
Having the same issue. It doesn't seem to be blocking anything, but it'd be nice if it could be fixed. [Especially since it apparently was back in October last year?](https://github.com/omab/python-social-auth/issues/755)

**Comment by tom-dalton-fanduel** [comment]
Created a PR with the propsed fix from above: #1008

**Comment by omab** [comment]
The fix for this issue got merged into the [social-app-django](https://github.com/python-social-auth/social-app-django) component.
